build.gradle

```diff
@@ -206,50 +217,52 @@ dependencies {
   compile group: 'org.springframework.boot', name: 'spring-boot-starter-aop', version: versions.springBoot
   compile group: 'org.springframework.boot', name: 'spring-boot-starter-validation', version: versions.springBoot
 
-  implementation("me.xdrop:fuzzywuzzy:1.3.1")
+  compile group: 'me.xdrop', name: 'fuzzywuzzy', version: '1.3.1'
   compile group: 'io.rest-assured', name: 'rest-assured', version: versions.restAssured
   compile group: 'io.rest-assured', name: 'json-path', version: versions.restAssured
   compile group: 'io.rest-assured', name: 'xml-path', version: versions.restAssured
   compile group: 'io.github.openfeign', name: 'feign-jackson', version: '11.0'
-  compile group: 'com.github.everit-org.json-schema', name: 'org.everit.json.schema', version: '1.14.0'
+  compile group: 'com.github.everit-org', name: 'json-schema', version: '1.14.1'
   compile group: 'javax.validation', name: 'validation-api', version: '2.0.1.Final'
   compile group: 'org.elasticsearch', name: 'elasticsearch', version: '7.16.3'
 
   compile group: 'io.springfox', name: 'springfox-swagger2', version: versions.springfoxSwagger
   compile group: 'io.springfox', name: 'springfox-swagger-ui', version: versions.springfoxSwagger
 
-  compile group: 'uk.gov.hmcts.reform', name: 'properties-volume-spring-boot-starter', version: '0.1.0'
+  compile group: 'com.github.hmcts', name: 'properties-volume-spring-boot-starter', version: '0.1.0'
   compile group: 'com.github.hmcts', name: 'java-logging', version: '5.1.7'
-  compile group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: '4.0.0'
+  compile group: 'com.github.hmcts', name: 'service-auth-provider-java-client', version: '4.0.3'
 
-  compile group: 'com.github.hmcts', name: 'sscs-common', version: '4.17.1'
+  compile group: 'com.github.hmcts', name: 'sscs-common', version: '4.17.1-RC-SSCS-10240'
```

Comment:
is this supposed to be in there?

Reply:
Yeah its the release candidate commons tag, it will be changed before deploy